src/ui/handler/touch_pan.js

```diff
@@ -54,7 +54,9 @@ export default class TouchPanHandler {
             }
         }
 
-        e.preventDefault();
+        if (e.cancelable) {
+          e.preventDefault();
```

Comment:
Nit: identation is 4 spaces:

```suggestion
            e.preventDefault();
```